deploy/gatekeeper.yaml

```diff
@@ -1,29 +1,123 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: gatekeeper-system
   labels:
-    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
```

Comment:
The `control-plane: controller-manager` label is not consistently applied across all GK resources. Missing from namespace and statefulset, but is added to the service. kubebuilder will add this label to the service: https://github.com/open-policy-agent/gatekeeper/blob/f591f52ecc4b373faf604e7722d809cefbfeac9e/pkg/webhook/policy.go#L95

I don't think we should remove this label as kubebuilder is generating the namespaceSelector for the webhook config that excludes all namespaces with this label.

https://github.com/open-policy-agent/gatekeeper/blob/f591f52ecc4b373faf604e7722d809cefbfeac9e/vendor/sigs.k8s.io/controller-runtime/pkg/webhook/bootstrap.go#L300-L308

Reply:
Good catch on the namespace, though I wonder if that label selector should be something different, more gatekeeper-specific?

In the interim, re-adding the label on the namespace.

The label remains applied to StatefulSet.